docs/learn/expert-guides/universal-profile/create-profile.md

```diff
@@ -66,23 +60,6 @@ For more details, see [LSP6 - Key Manager Standard](../../../standards/universal
 
 ![Universal Profile: overview of deployed contracts](../img/universal-profile-overview.jpeg)
 
-The [lsp-factory.js](../../../tools/lsp-factoryjs/getting-started.md) library will help us quickly deploy and set up a Universal Profile with just a few lines of code.
```

Comment:
Hmmm it is good to remove this big section, but what is then the relevancy and the goal of this guide then? To still deploy the UP + Key Manager manually without using LSP23?

Reply:
You are right, then let's remove this guide and redirect to the one with LSP23 instead.

Reply:
Maybe we can comment it out in the sidebar for now until we have a proper strategy for it.